objectstore/databasedumps.py

```diff
@@ -112,7 +112,7 @@ def remove_old_dumps(connection, container: str, days=None):
             objectstore.delete_object(connection, container, o_info)
 
 
-def download_database(connection, container: str, target: str=""):
+def download_database(connection, container: str, target: str = ""):
```

Comment:
Why the added spaces? PEP8 is without the spaces

Reply:
This is a part of flake8 cleanup.

Reply:
PEP8: "When combining an argument annotation with a default value, however, do use spaces around the = sign:" https://www.python.org/dev/peps/pep-0008/

Reply:
Ah I see, my mistake. Didn't realise it was different for annotated default arguments compared to unannotated default arguments. IDE formatting makes dumb ;)